src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -283,7 +283,7 @@ def load_arguments(self, _):
         c.argument('http_proxy_config')
         c.argument('enable_pod_security_policy', action='store_true')
         c.argument('enable_pod_identity', action='store_true')
-        c.argument('enable_workload_identity', arg_type=get_three_state_flag(), is_preview=True)
```

Comment:
Is this expected to remove "is_preview=True"?
If so, shall we also remove it in line 367?